apps/ledger-live-mobile/src/react-native-hw-transport-ble/index.js

```diff
@@ -1,46 +0,0 @@
-// @flow
```

Comment:
@Justkant Are we sure this should have been deleted? The tests are failing at the point of bluetooth stuff starting. CC @abdurrahman-ledger

Reply:
Adding the serviceUUID stuff back in